var/spack/repos/builtin/packages/r-lubridate/package.py

```diff
@@ -17,9 +17,9 @@ class RLubridate(RPackage):
     fun."""
 
     homepage = "https://cloud.r-project.org/package=lubridate"
```

Comment:
This is just a note that this could also be removed because the `cran` attribute would set the canonical URL.
```suggestion
    homepage = "https://haven.tidyverse.org/"
```